content/02.introduction.md

```diff
@@ -0,0 +1,13 @@
+## Introduction
+1. What is a preprint
+2. Why are preprints important?
+3. Mention how preprints are being integrated into scientist’s everyday workflow
+
+1. Talk about biorxiv and discuss how it is one of the repositories that maintains preprints
+2. Discuss obtaining biorxiv preprints was hard
+
+
+### Related Work

```

Comment:
Unless you're writing a computer science paper, this is not going to be a separate section. It would be worked into the abstract itself in narrative form where this contextual information makes sense.